src/scripts/calculation_download_gdrive_info.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import json
+import math
+import os
+import typing
+from urllib import request
+
+import pandas as pd
+
+
+def replace_nan(value, replace):
+    try:
+        if math.isnan(value):
+            return replace
+    except TypeError:
+        pass
+    return value
+
+
+def read_candidate_csv(file):
```

Comment:
The function names and variable names a good (descriptive) but I'm wondering whether or not we should have some docstrings in this file.

Reply:
That would be a good idea. I'll add some.